### PR TITLE
additional human drones

### PR DESCRIPTION
**Pull request by starmlerp**
This pull Request adds two more human drones and a ship to the Vanilla game: the Defense Drone, the Nock, and a Modified Freighter, a version of the Freighter able to carry a pair of drones.

## Defense Drone
The Defense Drone is a drone produced by Korunak Robotics, a sister company of the Korunak Photovoltaics, and is available from the start of the game. The drone has been balanced to be ridiculously cheap and nimble, for the sake of the pathetically low hull integrity and almost nonexistent firepower. This addresses #4034 in the opposite way: As this drone can be found in the most of the shipyards across the human space, it is meant to be very expendable quick missile sponge that will let the pilot jump away while the pirates are distracted.

## Nock
The Nock is a Syndicate produced drone, released to the market about the same time as the Barb fighter. Like other human drones, it is cheap and weak, although a bit slow. Unlike other human drones, it has a shield matrix, although no means of recharging it. With a still relatively low selling price, it is a pretty decent upgrade to the Defense Drone, awhile still affordable for the more tight budgeted pilots.

## Modified Freighter
The Modified Freighter is a Freighter modified to be able to carry a pair of drones. Alongside that, there are a couple of other upgrades, namely, the increased shields and hull, increased outfit and weapons space, a bit higher price, lower cargo capacity, and two extra required crew, to indicate that the drones are remotely controlled. I am also considering adding a single gun port to this model, in order to let players install a missile weapon, or a primary weapon, if they consider it manoveurable enough.

Since both of the drones are meant to be cheap and are not meant to last long in combat, both utilize KP-6 Photovoltaic Panels as their primary power source, making them more relyiant on their LP036a Battery packs.

## Controversy
It is a well known fact that human space is saturated with both outfits and ships of every category, leaving little reason to add more. However, such cannot be said for the drone category of ships. currently, in human space, there is only one drone, (currently) impossible to purchase in any of the shipyards without of the help of plugins or cheating. Since it is known that almost every other race has at least one drone for sale at some point in the game, it is surprising to find out that the species with the greatest surface area in the galaxy has only one (two, if you account for the Remnant) license prohibited drone. Since they are very affordable, usually provide as much of a distraction to the attacking pirates as a fighter, and require no crew, they would be an obvious go-to for merchants captains early in the game, and as such typical for humans.

## Plugin
These drones can be tested in [this](https://github.com/starmlerp/ESDefenseDronePlugin) plugin.

## Other

**This is still a draft PR, since the aforementioned ships might require balancing, and there are still quite a few bugs to fix, and features to implement.**

## Issues
So far, I've encountered two issues while preparing this PR:

**The Nock drones won't be re-deployed once their shields are fully charged.** This might be because of the ratio of shields over hull, and thus might require its separate bug report/feature request.

**All the features of this PR, including the ship designs, ship features, availability in shipyards, fleet composition, purpose and similar is a subject of change.**

All credits to @loiseer for the ship designs. This wouldn't have been possible without him.

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/140
- [ ] Fix the issue with the drones not re-deploying

## Testing Underway
As mentioned, the balancing is yet to be done for this PR.

### CODE REVIEW

**Comment by starmlerp** [comment]
Currently, the drone patrol fleets will only spawn in Rutilicus system. This is mainly for debugging purposes. This will be changed and fleets will be added to the other planets once the issue is addressed.

**Comment by Amazinite** [comment]
Still interested in this? Looks like there hasn't been any progress on it leaving draft since shortly after posting.

File: data/human/fleets.txt [hunk 1]
```diff
@@ -8,6 +8,32 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+fleet "Southern Patrol Drones"
```
Comment (Amazinite) [inline data/human/fleets.txt:11]: I'm not a fan of these fleets. Not only does this create scanners in southern space to catch a player's illegal cargo where normally there were not, but these also basically have the same purpose as the Flycatcher, which gets into the realm of using mechanics/ideas in the early game instead of saving them for later to make exploration more interesting (talked about in #4986). These fleets also run into the issue of systems that the player stays in with these fleets becoming filled to the brim with drones.

File: data/human/events.txt [hunk 2]
```diff
@@ -565,6 +566,58 @@ event "southern carriers 2"
...
+			"Modified Freighter (Fancy)"
+			"Nock" 2
+		add variant 5
+			"Modified Freighter (Fancy)"
+			"Nock (Modified)" 2
+	fleet "Small Core Merchants"
+		add variant 5
+			"Modified Freighter"
+			"Nock" 2
+			add variant 5
```
Comment (Amazinite) [inline data/human/events.txt:586]:
```suggestion
		add variant 5
```

**Comment by Zitchas** [comment]
The defense drone reminds me heavily of the flare launcher from EV: It's role is basically to be launched to absorb missile hits on behalf of the parent ship. A really expensive flare.

All in all, I like the idea of adding two new drones, as well as a adding a variant freighter with a pair of drone bays. Having merchants use said variant would be good; Not sure I like the idea of the drones being on patrol, though.

My recommendation would be to remove them from their patrols and just have them as a cheap drone that shows up on a few merchant ships, and leave it at that for now.

**Comment by starmlerp** [comment]
The idea of drone patrols in the southern space is to provide those governments with some sort of policing body, since we know for a fact that the navy is actively avoiding that part of space. Although it doesn't have much appeal in the terms of mechanics, it has a great lore appeal, with governments taking action into their own hands of protecting themselves. I might still remove them entirely, making this PR pretty much ready for merge.

**Comment by Zitchas** [comment]
Part of the reason I'm not fond of having the drone patrols is that it implies a greater degree of sophistication: A couple "dumb" drones that a freighter can launch and target as a distraction don't have to make any decisions: Just attack whatever their controller said to attack. Drones on patrol, on the other hand, have to have decision making capabilities to determine what (if anyone) they should be attacking, which party is in the wrong, etc. 

Despite what the player sees, I strongly doubt there is actually a "pirate" government that pirates conveniently set their IFF to in order to make it easy for patrols to know to attack them. Likewise, I strongly doubt there is a "merchant" government that ships identify themselves as to let patrols know they are to be protected.

**Comment by starmlerp** [comment]
Right. Off with patrols.

File: copyright [hunk 3]
```diff
@@ -719,6 +719,15 @@ Copyright: Lia Gerty (https://github.com/ravenshining)
...
  
+Files:
+ images/ship/defense_drone*
+ images/ship/mfreighter*
+ images/ship/nock*
+ images/thumbnail/defense_drone*
+ images/thumbnail/mfreighter*
+ images/thumbnail/nock*
+Copyright: Loyse
+License: CC-BY-SA-4.0
```
Comment (Amazinite) [inline copyright:730]: The drones were made from scratch, but the freighter wasn't.
```suggestion
Files:
 images/ship/defense_drone*
 images/ship/nock*
 images/thumbnail/defense_drone*
 images/thumbnail/nock*
Copyright: Loyse
License: CC-BY-SA-4.0

Files:
 images/ship/mfreighter*
 images/thumbnail/mfreighter*
Copyright: Loyse
License: CC-BY-SA-4.0
Derived from works by Michael Zahniser (under the same license)
```

File: data/human/events.txt (hunk 2 above)
Comment (Amazinite) [reply data/human/events.txt:586]: This one is still an issue.

**Comment by Amazinite** [comment]
I'm still not interested in this for reasons already mentioned here, on stream, and on Discord. 
TL;DR
* Human space is already bloated, why bloat it more? 
* There exist lore reasons as to why civilian drones might not exist in Endless Sky at the time that the story takes place.
* The idea of using drones to distract pirates is just a more expensive method compared to dumping cargo to distract pirates.
* I don't think that making carried ships super expendable is the way to go about making them more appealing to use, because losing a ship is still losing a ship no matter how cheap it is.
* Probably a few other reason I can't think of right now.

There is still a need for the Syndicate to have a drone for the Class C freighter, though. (Either that or we change the Class C Freighter to use fighters, but for story reasons I'm not too interested in that.) As such, Mlerp has mentioned opening a new PR with just one of the drones to be unlocked during a Syndicate campaign. Given that though, this PR won't be going anywhere, so I'm just going to close it.
